test/vstest/runtests.cmd

```diff
@@ -0,0 +1,7 @@
+set target=%1
+set version=%2
+powershell.exe -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "./dotnet-install.ps1 -Version %version% -Channel 2.0 -NoCdn -InstallDir %HELIX_CORRELATION_PAYLOAD%"
```

Comment:
```suggestion
powershell.exe -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "Invoke-WebRequest https://get.dot.net/v1/dotnet-install.ps1 -o ./dotnet-install.ps1"
powershell.exe -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "./dotnet-install.ps1 -Version %version% -Channel 2.0 -NoCdn -InstallDir %HELIX_CORRELATION_PAYLOAD%"
```
An alternative to having a copy of dotnet-install.ps1 in the payload (although its size is negligible).